### DISCUSSION TITLE
OSGi specs and equinox implementation

### DISCUSSION THREAD

**Discussion by salvatore-coppola** [eclipse-equinox/equinox#379] [General]
Hi,

Is there a table indicating, for each version of Equinox, the version of the OSGi specification implemented?

**Comment by laeubi** [comment]
Maybe its worth to explain for what this is needed, from a technical point of view you can derive it from the jar but usually using simply the latest version will give you the latest OSGi specification what is backward compatible to older specification releases.

**Reply by salvatore-coppola** [reply]
hi @laeubi, thanks for the quick reply we are using righit now the equinox version 4.18 and I'd like to check the osgi specs implemented, do you know where I can find it ?

**Reply by laeubi** [reply]
You mean eclipse release? Because latest equinox is currently version 3.18: https://github.com/eclipse-equinox/equinox/blob/master/bundles/org.eclipse.osgi/

**Reply by salvatore-coppola** [reply]
i mean this release https://archive.eclipse.org/equinox/drops/R-4.18-202012021800/index.php
